### DISCUSSION TITLE
Support for Sonoff S31

### DISCUSSION THREAD

**Discussion by dk307** [RavenSystem/esp-homekit-devices#1790] [General]
Is [Sonoff S31](https://sonoff.tech/product/smart-plug/s31-s31lite/) supported by RavenSystem ? S31 has CSE7766 power monitoring which I don't find in the [Chip Type](https://github.com/RavenSystem/esp-homekit-devices/wiki/Power-Measure#Chip-Type) for Power Monitoring

Thanks
Deepak

**Comment by tortengraeber** [comment]
I need the same info for my Refoss P11 Outlet´s

**Comment by shadowgrapher** [comment]
Looks like it's already [supported](https://github.com/RavenSystem/esp-homekit-devices/discussions/2102#discussioncomment-6884863):
>– Hi is there any chance of adding HLW8032 / CSE7766 support?
>– You must use Free Monitor service to read data from those chips.
